payment/payment.go

```diff
@@ -189,7 +189,7 @@ func (p *Payment) ScriptHash() (string, error) {
 }
 
 // ConfidentialScriptHash is a method of the Payment struct to derive a
-//base58 confidential p2sh address
+//blech32 confidential p2sh address
```

Comment:
here it's correct the comment `base58` as was before

Reply:
Done